### PR TITLE
Always execute the latest brightness change request instead of queuing

### PR DESCRIPTION
**Pull request by usr-sse2**

### CODE REVIEW

**Comment by usr-sse2** [comment]
I wanted not to remove `sin` but to calculate all necessary values once and save them into an array.
I think we don't need to support devices with more than one built-in LVDS display.

**Comment by vit9696** [comment]
Right, perhaps we can put the precalculation to a separate PR and merge this code as is for now? Could you update the title and include the changelog changes we need to apply after merging?

File: WhateverGreen/kern_igfx_backlight.cpp
Comment (vit9696) [inline WhateverGreen/kern_igfx_backlight.cpp:377]: Use Buffer::create<double>, in my opinion. It is specifically for that purpose.

File: WhateverGreen/kern_igfx_backlight.cpp
Comment (vit9696) [inline WhateverGreen/kern_igfx_backlight.cpp:269]: Leave a comment that 1 is intended here.

**Comment by usr-sse2** [comment]
Don't merge this now.
1. The formula is incorrect, should be (1-cos(π*i/steps))/2 so that the derivative is 0 at both ends of the interval.
2. Even with this replacement, this function is worse than linear when a "Brightness Up" or "Brightness Down" key is pressed  and held, so it seems that it's better to revert to linear function.

File: WhateverGreen/kern_igfx_backlight.cpp
Comment (usr-sse2) [reply WhateverGreen/kern_igfx_backlight.cpp]: Removed

File: WhateverGreen/kern_igfx_backlight.cpp
Comment (usr-sse2) [reply WhateverGreen/kern_igfx_backlight.cpp:276]: Done

**Review by Craze1666 — APPROVED** [review]

**Pull request merged**
